feat(components/indicators): update inline help emitter type to void

Blackbaud-ErikaMcVey opened the pull request.

Blackbaud-BobbyEarl commented on webdrivertest/test/common.js:
```diff
@@ -15,6 +15,35 @@
+                    }
+                    logError('See: ' + violation.helpUrl + '\n');
+                }
+                expect(ret.value.violations.length).toBe(0, ' number of accesiblity violations');
```
"Accessibility" is misspelled.

Blackbaud-BobbyEarl commented:
Approved

blackbaud-sky-build-user commented:
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/584/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/popovers-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/584/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/584/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/584/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/584/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/584/playground/)

Blackbaud-ErikaMcVey committed:
feat(components/indicators): update inline help emitter type to void

This change updates the inline help component's `actionClick` event emitter type to void instead of
any, since it never emits a value.
